binder/environment.yml

```diff
@@ -9,5 +9,10 @@ dependencies:
   # ome: minimal dependencies required to use the Python API
   - zeroc-ice36-python
   - omero-py
-
-
+  # dependencies to use for zarr work
+  - dask
+  - dask-image
```

Comment:
I wasn't aware of this, is it a potential location for the zarr-image-py library @joshmoore ?

Reply:
I used ``dask-image`` for image manipulations
e.g.
```dask_image.ndfilters.gaussian_filter(plane, sigma=[1, 1])``

Reply:
@manics : possibly. Needs more discussion with them.